internal/local/ethernet.go

```diff
@@ -171,23 +279,29 @@ func (service *ProvisioningService) AddCerts(handles Handles, ieee8021xConfig co
 	if ieee8021xConfig.PrivateKey != "" {
 		handles.privateKeyHandle, err = service.interfacedWsmanMessage.AddPrivateKey(ieee8021xConfig.PrivateKey)
 		service.handlesWithCerts[handles.privateKeyHandle] = ieee8021xConfig.PrivateKey
-		if err != nil {
+		if err != nil && strings.Contains(err.Error(), "already exists") {
+			// handles.privateKeyHandle, _ = service.GetPrivKeyHandle(ieee8021xConfig.PrivateKey)
```

Comment:
Commented out because it was not used, but may be useful for EA. The function that is used is a new function is also commented out because it does not yet work as intended